app/controllers/ems_common.rb

```diff
@@ -678,6 +720,70 @@ def refreshemss
     end
   end
 
+  def call_ems_pause(emss)
+    process_emss(emss, "pause_ems") unless emss.empty?
+    return if @flash_array.present?
+
+    add_flash(n_("Disable initiated for %{count} %{model} from the %{product} Database",
+                 "Disable initiated for %{count} %{models} from the %{product} Database", emss.length) %
+                {:count   => emss.length,
+                 :product => I18n.t('product.name'),
+                 :model   => ui_lookup(:table => table_name),
+                 :models  => ui_lookup(:tables => table_name)})
+  end
+
+  def call_ems_resume(emss)
+    process_emss(emss, "resume_ems") unless emss.empty?
+    return if @flash_array.present?
+
+    add_flash(n_("Resume initiated for %{count} %{model} from the %{product} Database",
+                 "Resume initiated for %{count} %{models} from the %{product} Database", emss.length) %
+                {:count   => emss.length,
+                 :product => I18n.t('product.name'),
+                 :model   => ui_lookup(:table => table_name),
+                 :models  => ui_lookup(:tables => table_name)})
+  end
+
+  def pauseemss
+    assert_privileges(params[:pressed])
+    if @lastaction == "show_list"
+      emss = find_checked_items
+      if emss.empty?
+        add_flash(_("No %{model} were selected for pause") % {:model => ui_lookup(:table => table_name)}, :error)
+      end
+      call_ems_pause(emss)
+      show_list
+      @refresh_partial = "layouts/gtl"
+    else
+      if params[:id].nil? || model.find_by_id(params[:id]).nil?
+        add_flash(_("%{record} no longer exists") % {:record => ui_lookup(:table => table_name)}, :error)
+      else
+        call_ems_pause([params[:id]])
+      end
+      params[:display] = @display
+    end
+  end
+
+  def resumeemss
```

Comment:
This is extremely similar to `pauseems` method.  Consider DRYing to one method with a param?